unity-client/Assets/Scenes/InitialScene.unity

```diff
@@ -1257,7 +1257,7 @@ PrefabInstance:
         type: 3}
       propertyPath: mouseCatcher
       value: 
-      objectReference: {fileID: 0}
```

Comment:
is this scene change needed?

Reply:
Yes, this is a reference that is missing for the builder in world. There is another PR that add unit test to cover that use case